lib/catalog/martyrology.ts

```diff
@@ -802,6 +830,12 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Clare',
       titles: [Titles.Virgin],
     },
+    claudine_thevenet_religious: {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Claudine Thévenet',
+      dateOfDeath: 1837,
+      titles: [Titles.Religious],
```

Comment:
Add religious name to the key name.

According to [Wikipedia](https://en.wikipedia.org/wiki/Claudine_Th%C3%A9venet), she was a foundress; is she not celebrated with this title in the Archdiocese of Lyon?

```suggestion
    marie_of_saint_ignatius_claudine_thevenet_religious: {
      canonizationLevel: CanonizationLevels.Saint,
      name: 'Claudine Thévenet',
      dateOfDeath: 1837,
      titles: [Titles.Religious],
```

Reply:
In the proper missal, only "Religious" is specified for her title. 
About being a foundress is only mentioned in her short biography in the missal.